lake/oneshot_cls.py

```diff
@@ -166,7 +165,7 @@ def main():
           with torch.no_grad():
             for batch_idx_val, (val_data, val_target) in enumerate(val_loader):
 
-              if 0 > MAX_VAL_STEPS > batch_idx_val:
+              if 0 > MAX_VAL_STEPS > batch_idx_val:   ##### y esto pa que!
```

Comment:
I don't see how this would ever work with positive numbers? 🤷

Reply:
I think the idea behind it is to give the option to limit the number of validation steps. For example, if the validation set has 10,000 samples with a `batch_size` of say `100`, by default you would be doing `100` validation steps in total. 

If you only want to validate on a portion of that (i.e. end it early), you can break the loop by checking against the `MAX_VAL_STEPS`.

I think to make this work it, the condition needs to be: `batch_idx_val >= MAX_VAL_STEPS`?

Reply:
Agree.